src/main/resources/api/api.yaml

```diff
@@ -250,6 +404,75 @@ definitions:
   #
   # Questions
   #
+  V1Question:
+    type: object
+    required:
+      - text
+      - type
+      - topic
+      - answers
+    properties:
+      id:
+        type: integer
+        format: int64
+      text:
+        type: string
+      type:
+        type: integer
+        description: question type means the type and quantity of answers
+        enum:
+        - 1
+        - 2
+        - 3
+        - 4
+      topic:
+        $ref: '#/definitions/V1Identity'
+      answers:
+        $ref: '#/definitions/V1AdminAnswers'
+    example:
+      id: 100500
+      text: 'Select private networks:'
+      type: 1
+      topic: 1
+      answers:
+        $ref: '#/definitions/V1AdminAnswers'
+  V1Questions:
+    type: array
+    items:
+      $ref: '#/definitions/V1Question'
+  #
+  # Answers
+  #
+  V1AdminAnswer:
+    type: object
+    required:
+      - text
+      - question
+      - isRight
+      - number
+    properties:
+      id:
+        type: integer
+        format: int64
+      text:
+        type: string
+      isRight:
+        type: boolean
+      number:
+        type: integer
+        format: int64
+      question:
```

Comment:
I'm thinking about question reference here, and because we always use answer in question context (there is no separate answer API), we could remove question from answer al all.